spec/committer/commit_generator_spec.rb

```diff
@@ -4,7 +4,7 @@
 require 'committer/commit_generator'
 
 RSpec.describe Committer::CommitGenerator do
-  let(:diff) { load_fixture('sample_diff.txt') }
```

Comment:
i think sample diff is only used here so if you removed it we can delete the file